modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,30 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
+
+Being able to view a saved state of a project is useful if you want to:
+
+* Undo an inadvertent change you made to a project and revert back to the working state.
+* Review changes made by other users on the project.
+* Retrieve older values of some project configurations.
+
+
+== Procedure
+
+To restore to a previous version:
+
+. Click *History* on the upper left of the canvas to open the Project History list on the right of the canvas. 
+. Click a saved state from the list.
+. Inspect the saved state in the sidebar by clicking on an object in the flow to ensure it is the correct one to restore.
+. Click *Restore* to restore to the state you selected. +
```

Comment:
To restore what? To restore the state you selected? Restore takes an object.
Also, the title of the procedure is "To restore a previous version of your project". This step uses "state", not "version".

Reply:
You are correct, I am mixing up the terms here.